user/user.py

```diff
@@ -313,6 +322,10 @@ class UserinfoEmailMismatchError(Exception):
                 True
             )
 
+    except StateMismatchError:
+        # Invalid state parameter.
+        log_error("Invalid state parameter")
```

Comment:
Can we add the user email in the log message, so that it is easier to see for ops to which user this relates?